<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_sieciowy</class>
 <widget class="QDialog" name="dialog_sieciowy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWlaczSiec">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Włącz sieć</string>
   </property>
  </widget>
  <widget class="QLabel" name="kontrolkaStatusu">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupTryb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>271</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioSerwer">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tryb Serwer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioKlient">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tryb Klient</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="linePortSerwer">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStartSerwer">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Włącz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStopSerwer">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyłącz</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineIP">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePortKlient">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPolaczKlient">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Połącz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRozlaczKlient">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Rozłącz</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>431</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="kontrolkaStatus">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialog_sieciowy</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialog_sieciowy</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
